papers/eutxo/Makefile

```diff
@@ -0,0 +1,10 @@
+MAIN=eutxo
+
+$(MAIN).pdf:
+	xelatex $(MAIN).tex
```

Comment:
I don't know how much this helps them, since most people don't use `xelatex` ;) But it gives the idea, I guess.

Reply:
I will replace it with `pdflatex`, but I do not like the font thickness that much (it's quite less boldy).

Reply:
I don't think it matters: they're going to recompile it, and the one that's hosted by us is built with `pdflatex` anyway.

Reply:
Actually, the Makefile is useless in this repo, I will just remove it.